pkg/plural/api/authorize_proxy_user.go

```diff
@@ -37,3 +40,20 @@ func AuthorizeProxyUser(ctx context.Context, token, clusterId, pluralURL string)
 		},
 	}, nil
 }
+
+func CreateAuditLogInBackground(log *zap.Logger, agentId int64, r *http.Request, token, clusterId, pluralURL string) {
+	go func() {
+		log = log.With(logz.AgentId(agentId))
+
+		client := plural.New(pluralURL, token)
+		_, err := client.Console.AddClusterAuditLog(context.Background(), console.ClusterAuditAttributes{
```

Comment:
Is this going to pass the user who acted through?

Reply:
Ideally we just use the auth information from the original request to auth to the console api for tha